### PR TITLE
Fix a few spelling issues across the knowledge base :book:

### PR DESCRIPTION
**Pull request by jacob-meacham**
Ran all of the md files through aspell

### CODE REVIEW

**Comment by yoshuawuyts** [comment]
hahaha, this great! thanks heaps hey

File: arch/auth.md [hunk 1]
```diff
@@ -212,10 +212,10 @@ XSS bugs!
 - disallow any non-JSON POST/PUT/PATCH/DELETE
 - `multipart/form-data` needs to still be handled
 
-### check HTTP referer header
-- check `referer` header for `multipart/form-data` POST and friends
-- `referer` is set by a browser and tells which page (url) triggered a request
-- disallow requests without `referer` present
+### check HTTP referrer header
+- check `referee` header for `multipart/form-data` POST and friends
```
Comment (yoshuawuyts) [inline arch/auth.md]: hah

File: programming/maintenance.md [hunk 2]
```diff
@@ -24,4 +24,4 @@ Code rot.
 [ tbi ]
 
 ## See Also
-- [a codebae is an organism](http://www.meltingasphalt.com/a-codebase-is-an-organism/)
```
Comment (yoshuawuyts) [inline programming/maintenance.md]: lol, `codebae`

**Comment by yoshuawuyts** [comment]
merged, thanks again!

File: programming/maintenance.md (hunk 2 above)
Comment (jacob-meacham) [inline programming/maintenance.md]: haha yeah, this was my favorite typo as well!

**Pull request closed without merging**